src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,111 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int CAPACITY = 16;
+    private int size;
+    private Node<K, V>[] table;
+
+    public MyHashMap() {
+        table = new Node[CAPACITY];
+    }
 
     @Override
     public void put(K key, V value) {
-
+        if (size >= table.length * LOAD_FACTOR) {
+            resize();
+        }
+        int index = getHash(key);
```

Comment:
```suggestion
        int index = getIndexForKey(key);
```
Hash and index are different things